src/com/company/Main.java

```diff
@@ -6,9 +6,59 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    public static Address createAddress(final Scanner scanner) {
+        System.out.println("Enter the address");
+        System.out.println("City:");
+        final String cityNewBank = scanner.next();
+        System.out.println("Zip code:");
+        final int zipCodeNewBank = scanner.nextInt();
+        System.out.println("Street:");
+        final String streetNewBank = scanner.nextLine();
+        System.out.println("House number:");
+        final int houseNumberNewBank = scanner.nextInt();
+        System.out.println("Apartment number:");
+        final int apartmentNumberNewBank = scanner.nextInt();
+        return new Address(cityNewBank, zipCodeNewBank, streetNewBank, houseNumberNewBank, apartmentNumberNewBank);
+    }
+
+    public static void createBank(final Scanner scanner, final List<Bank> banks) {
+        System.out.println("Create new bank");
+        final String nameBank = scanner.nextLine();
+        createAddress(scanner);
+        final BankBranch bankBranch = new BankBranch(createAddress(scanner));
+        final Bank bank = new Bank(nameBank, bankBranch);
+
+        banks.add(bank);
+    }
+
+    public static void listBank(final List<Bank> banks) {
+        System.out.println("List banks");
+        for (int i = 0; i < banks.size(); i++) {
+            System.out.println(i);
+            System.out.println(banks.get(i));
+        }
+    }
 
-        List<Bank> banks = new ArrayList<>();
+    public static void selectBank(final Scanner scanner, final List<Bank> banks) {
+        System.out.println("Select bank");
+        final int select = scanner.nextInt();
+        final Bank bankSelect = banks.get(select);
+        System.out.println(bankSelect);
+    }
+
+    public static void deleteBank(final Scanner scanner, final List<Bank> banks) {
```

Comment:
It'd be better to return boolean in dependency of removal result. If it went ok, `true`, `false` otherwise.